pkg/providers/common/auditpolicy.go

```diff
@@ -0,0 +1,252 @@
+package common
+
+import (
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+	"sigs.k8s.io/yaml"
+
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+)
+
+func GetAuditPolicyYaml(kubeVersion anywherev1.KubernetesVersion) ([]byte, error) {
+	return yaml.Marshal(GetAuditPolicy(kubeVersion))
+}
+
+func GetAuditPolicy(kubeVersion anywherev1.KubernetesVersion) *auditv1.Policy {
+	a := baseAuditPolicy()
+	if kubeVersion == anywherev1.Kube124 {
```

Comment:
Since v1beta1 audit API version has been removed in 1.24, can we generalize this to anything >= 1.24? Maybe using a custom comparator on KubeVersion?